File: 4.Clients/WebApi/ClientApp/src/entities/task.ts
import { TaskItem } from "./taskItem";
import { Consultant } from "./consultant";

export class Task {
    id:number;    
    title: string;
    isApprove: boolean;
    isNew: boolean;
    creationDate: Date;
    endDate: Date;
    
    consultantId: number;
    consultant: Consultant;

    taskItems: TaskItem[]; 
}

File: 1.Domain/Domain.Model/Task.cs
using Core;
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain.Model
{
    public class Task: Entity<int>
    {
        public string Title { get; set; }
        
        public bool IsApprove { get; set; }
        public bool IsNew { get; set; }

        public DateTime CreationDate { get; set; }
        public DateTime EndDate { get; set; }

        public int ConsultantId { get; set; }
        public Consultant Consultant { get; set; }

        public IList<TaskItem> TaskItems { get; set; }
    }
}
